modules/tensor_mechanics/src/actions/CavityPressureUOAction.C

```diff
@@ -51,10 +31,7 @@ CavityPressureUOAction::act()
 {
   InputParameters params = _factory.getValidParams("CavityPressureUserObject");
 
-  params.applyParameters(parameters(), {"initial_temperature"});
-
-  if (isParamValid("initial_temperature"))
-    params.set<Real>("initial_temperature") = getParam<Real>("initial_temperature");
+  params.applyParameters(parameters());
```

Comment:
I'm pretty sure applyParameters checks if the input parameter is valid before applying and that this change is fair, is that the case...maybe @dschwen?

Reply:
Yes